ansible/config_sonic_basedon_testbed.yml

```diff
@@ -74,6 +74,101 @@
         vm_base: "{% if testbed_facts['vm_base'] != '' %}{{ testbed_facts['vm_base'] }}{% else %}''{% endif %}"
     when: testbed_name is defined
 
+  - name: Run L1 Switch connection panel for RDMA testing
+    block:
+    - name: Get HWSKU type
+      set_fact:
+        hwsku_type: "{{ hostvars[inventory_hostname]['hwsku'] }}"
+
+    - name: Create connections for 8111 testbed
+      block:
+      - name: Create 1st port connection for Cisco 8111 testbed
+        raw: connection create p40 to p45
```

Comment:
`p40` and `p45` are hard coded.

What if the physical connection is changed?

My suggestion is to define the physical connection in some connection graph csv file.
If the current connection graph csv schema cannot meet the requirement, maybe need to define a new schema enhance connection graph to support some new schema for L1 switch.